.github/workflows/updatecli.yml

```diff
@@ -19,11 +48,42 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pipeline-name: [beats, golang, hermit]
+        pipeline-name: [ beats, golang, hermit ]
     steps:
       - uses: actions/checkout@v4
       - name: Init Hermit
         run: ./bin/hermit env -r >> $GITHUB_ENV
+      - name: Set Branch
+        run: echo "GIT_BRANCH=main" >> $GITHUB_ENV
```

Comment:
You can just use the `env:` directive in the next step

Reply:
I see, I followed a thread on stackoverflow that mislead me a bit https://docs.github.com/en/actions/learn-github-actions/variables, I'll update!